pwiz_tools/Skyline/TestFunctional/DdaSearchTest.cs

```diff
@@ -1,50 +1,50 @@
-﻿/*
- * Original author: Matt Chambers <matt.chambers42 .at. gmail.com >
- *
- * Copyright 2020 University of Washington - Seattle, WA
- * 
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.IO;
-using System.Linq;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using pwiz.Common.Chemistry;
-using pwiz.Skyline.Alerts;
-using pwiz.Skyline.Controls;
-using pwiz.Skyline.FileUI;
-using pwiz.Skyline.FileUI.PeptideSearch;
-using pwiz.Skyline.Model.DdaSearch;
-using pwiz.Skyline.Model.DocSettings;
-using pwiz.Skyline.Model.Results;
-using pwiz.Skyline.Properties;
-using pwiz.Skyline.SettingsUI;
-using pwiz.Skyline.Util;
-using pwiz.SkylineTestUtil;
-
-namespace pwiz.SkylineTestFunctional
-{
-    [TestClass]
-    public class DdaSearchTest : AbstractFunctionalTest
-    {
-        public struct ExpectedResults
-        {
-            public int proteinCount;
-            public int peptideCount;
-            public int precursorCount;
+﻿/*
+ * Original author: Matt Chambers <matt.chambers42 .at. gmail.com >

```

Comment:
It's hard to tell what changed here. Line endings again?

Reply:
Yeah, that file was a mixed bag of line endings, and I got tired of being asked about it.

Reply:
(You can tell the github diff to ignore line endings, BTW)